src/commands/contract/deploy.ts

```diff
@@ -70,13 +68,31 @@ export class DeployContract extends SwankyCommand<typeof DeployContract> {
       );
     }
 
+    if(!flags.account && this.swankyConfig.defaultAccount === null) {
```

Comment:
Keep those checks DRY with a dedicated helper to avoid code duplication in [contractCalls.ts](src/lib/contractCall.ts)